client/scss/utilities/_root-scope-classes.scss

```diff
@@ -32,10 +32,14 @@
   display: inline-flex;
 }
 
-.flex--center {
+.flex--v-center {
```

Comment:
Hmmmm – it requires that the element it's assigned to is `display: flex` (which is available through the `.flex` selector, so it makes some sense.